layers/+tools/tree-sitter/config.el

```diff
@@ -31,3 +31,7 @@
 
 (defvar tree-sitter-fold-indicators-enable t
   "If non nil, and `tree-sitter-fold-enable' is non nil, show fold indicators in fringe.")
+
+(defvar spacemacs-tree-sitter-hl-black-list
+  '()
+  "List of major modes not allow tree sitter highlight mode.")
```

Comment:
Just use `nil` instead of `'()`

```elisp
(defvar spacemacs-tree-sitter-hl-black-list nil
  "List of major modes not allow tree sitter highlight mode.")
```